WaltonSustainabilityCentre-Requirements.txt

```diff
@@ -48,9 +48,9 @@ Originator: Karen Black, Centre Administrator
 Comments: GS
 
 REQ-ID: 010
-Description: The product shall identify additional equipment requirements
+Description: The product shall predict stock levels of regularly used resources, and notify staff when re-stocking is required.
```

Comment:
I think we should have "EPIC" tasks like high level overview of minimum set of requirements to fulfil idea of project. What you are suggesting is specifying this requirement more, which in fact does not help at all. 

I suggest to have generic tasks like:
- Track equipment for events

Which as you suggested should contain in future subtasks of prediction, notifications etc..